app/mainwindow.cpp

```diff
@@ -557,8 +581,11 @@ void MainWindow::styleCustomStyle()
     QAction *action = qobject_cast<QAction*>(sender());
 
     generator->setCodeHighlightingStyle("default");
-
-    ui->plainTextEdit->loadStyleFromStylesheet(":/theme/default.txt");
+    if(options->isSourceAtOneSizeEnabled()) {
+        ui->plainTextEdit->loadStyleFromStylesheet(":/theme/default-.txt");
```

Comment:
In order to stay consistent, please rename the current one to default+.txt and introduce the new one as default.txt.